sdk/src/test/java/com/google/cloud/dataflow/sdk/runners/dataflow/DataflowUnboundedReadFromBoundedSourceTest.java

```diff
@@ -268,6 +268,53 @@ public void testReadFromCheckpointBeforeStart() throws Exception {
     unboundedSource.createReader(options, checkpoint).getCurrent();
   }
 
+  @Test
+  public void testKind() {
+    DataflowUnboundedReadFromBoundedSource<?> read = new
+        DataflowUnboundedReadFromBoundedSource<>(new NoopNamedSource());
+
+    assertEquals("Read(NoopNamedSource)", read.getKindString());
```

Comment:
Port your static import of Assert.assertEquals back into Apache Beam to make diffing of files simpler